File: X/rust/XRPL_UKL1/soal1.rs
/*
Soal        : 1
a           : 5
b           : 9
Suku akhir  : 21
*/

fn main(){
    let a=5;
    let b=9;
    let suku_akhir=2;
    let mut u=a;
    let mut s=a;
    println!("Deret aritmatikanya adalah");
    for _ in 0..suku_akhir{
        println!("Angka U awal = {}", u);
        println!("Angka S awal = {}", s); 
        u+=b;
        s+=u;   
        println!("Angka U awal = {}", u);
        println!("Angka S awal = {}", s); 
    }
    println!("S = s - u = {} - {} = {}", s, u, (s-u));
    let total=s-u;
    println!("Jumlah Deret aritmatikanya adalah {}", total);
}

File: X/java/XRPL_UKL1/soal1.java
/*
Soal        : 1
A           : 5
B           : 9
Suku akhir  : 21
*/

public class soal1 {
   public static void main(String[] args) {
      int A=5,
          B=9,
          suku_akhir=2,
          u=A,
          s=A;
      System.out.println("Deret Aritmatikanya adalah"); 
      for(int i=0;i<suku_akhir;i++){                            
        System.out.println("Angka U awal = "+u);
        System.out.println("Angka S awal = "+s); 
        u+=B; 
        s+=u;    
        System.out.println("Angka U akhir = "+u);
        System.out.println("Angka S akhir = "+s);  
      }
      System.out.println("S = s-u = " + s + " - " +u + " = "+ (s-u)); 
      int total=s-u;  
      System.out.println("Jumlah Deret Aritmatikanya adalah "+total);
    }
}